### DISCUSSION TITLE
建议鼠标手势能增加方向识别。

### DISCUSSION THREAD

**Discussion by wj55** [shuax/MouseInc.Settings#280] [Ideas]
比如“DOWN-UP"这个鼠标手势。再"UP"完成后，能区分左右方向就好了。”UP“向左时，是一个鼠标手势。”UP"向右时，又是一个鼠标手势。能大大增加鼠标手势的相关快捷操作。
       现在软件在识别“DOWN-UP"这个鼠标手势时，不管“UP”向左，还是向右，都是同一个快捷操作，太浪费了。希望作者能考虑一下。

**Comment by shuax** [comment]
手势形状可以自己加的，但是太过相似的手势容易识别错误。
